docs/README.md

```diff
@@ -0,0 +1,112 @@
+[![Bulwark Logo](/docs/assets/bulwark-hero.png)](https://bulwark.security/)
+
+---
+
+Automated security decision-making under uncertainty.
+
+## 🏰 What is Bulwark?
+
+Bulwark is a fast, modern, open-source web application firewall and API security gateway that simplifies the
+implementation of detective security controls while offering comprehensive visibility into your web services. Bulwark's
+detection-as-code approach to rule definition offers security teams higher confidence in their response to persistent
+and adaptive threats. Bulwark plugins offer a wide range of capabilities, enabling security teams to define and evolve
+detections rapidly, without making changes to the underlying application.
+
+- **Open Source.** Freely available under a permissive Apache 2.0 license. We're committed to keeping it that way.
+- **Extensible.** Define custom detection logic using the SDK, or customize reusable parameterized detections to
+    your needs, without writing code.
+- **Observable.** Gain comprehensive visibility into both your traffic and the operation of your detections with built-in
+    observability features. Enrich your traffic data for improved context.
+- **Flexible.** Bulwark's plugin API enables detections to interact with Redis state and other services, including
+    machine learning models. Plugins can perform their detections collaboratively using Bulwark's ensemble decisions.
+    Securely decrypt session cookies to operate on application-level information.
+- **Testable.** Detections can have automated tests. Prevent misclassifications from making it to production.
+- **Accurate.** Bulwark has built-in mechanisms to help tune detections for high accuracy, minimize false
+    positives, and quickly remediate accuracy issues if they occur. Meet compliance requirements for detective controls
+    while avoiding false positives that would disrupt operations.
+- **Sandboxed.** Every detection runs inside a secure [WebAssembly](https://webassembly.org/) sandbox, isolating
+    detection logic, and ensuring that access never exceeds its permissions grants.
+- **Safe.** Deploy in observe-only mode and build confidence in the system before enabling request blocking.
+- **Commercially Supported.** Bulwark is actively developed and supported by
+    [Bulwark Security PBC](https://bulwark.security/). We intend to offer a freely available
+    [community ruleset](https://github.com/bulwark-security/bulwark-community-ruleset), an
+    advanced commercial ruleset, paid support, and professional services, in addition to a Bulwark cloud admin
+    application. Contact us ([sales@bulwark.security](mailto:sales@bulwark.security)) if you are interested in
+    discounted early access on a design partner basis for any of these commercial offerings.
```

Comment:
One thing I wasn't sure about was whether "design partner" needed a hyperlink out to a good explanation or if we could just assume people would Google it if they didn't know what it means. Unfortunately, I only found explanations written for the vendor perspective rather than from the design partner's perspective (the audience of this document), but maybe that's okay?